tests/metrics/test_metric_lightning.py

```diff
@@ -1,11 +1,13 @@
 import torch
+from torchmetrics import Metric as TMetric
 
 from pytorch_lightning import Trainer
-from pytorch_lightning.metrics import Metric, MetricCollection
+from pytorch_lightning.metrics import Metric as PLMetric
+from pytorch_lightning.metrics import MetricCollection
```

Comment:
@tchaton van you pls double check that this is sufficient compatibility test for PL and TM metrics